Ej7.rb

```diff
@@ -0,0 +1,20 @@
+puts 'Welcome'
+puts 'Algorithm that allows converting numerical grades to letters'
+puts 'Enter any grade of 1 to 20'
+
+grade=gets.to_i
+
+print 'Note is'
+
+if grade>=19 && grade<=20
```

Comment:
Ruby has a very convenient method called Between. Not every language has it but Ruby just rocks and make our life easier. You can check it out here.
[Rubydoc](https://ruby-doc.org/core-2.4.0/Comparable.html#method-i-between-3F)